tree/ntuple/v7/test/ntuple_basics.cxx

```diff
@@ -758,3 +758,23 @@ TEST(REntry, Basics)
    e->BindRawPtr("pt", &pt);
    EXPECT_EQ(&pt, e->GetPtr<void>("pt").get());
 }
+
+TEST(RFieldBase, CreateObject)
+{
+   auto ptrInt = RField<int>("name").CreateObject<int>();
+   EXPECT_EQ(0, *ptrInt);
+
+   void *rawPtr = RField<int>("name").CreateObject<void>().release();
+   EXPECT_EQ(0, *static_cast<int *>(rawPtr));
+
+   {
+      ROOT::TestSupport::CheckDiagsRAII diagRAII;
+      diagRAII.requiredDiag(kWarning, "[ROOT.NTuple]", "possibly leaking", false /* matchFullMessage */);
+      auto p = RField<int>("name").CreateObject<void>();
```

Comment:
Probably here as well, something like
```suggestion
      auto p = RField<int>("name").CreateObject<void>();
      delete static_cast<int *>(p.get());
```
(to avoid `release()`ing)